bluebottle/projects/models.py

```diff
@@ -718,15 +716,6 @@ def project_post_init(sender, instance, **kwargs):
 @receiver(post_save, sender=Project,
           dispatch_uid="bluebottle.projects.Project.post_save")
 def project_post_save(sender, instance, **kwargs):
-
-    if instance.payout_status == 'approved':
-        adapter = DoradoPayoutAdapter(instance)
-        try:
-            adapter.trigger_payout()
-        except ImproperlyConfigured:
-            logger.warning('Dorado not configured when project payout approved',
-                           exc_info=1)
```

Comment:
Why is this logger removed? Maybe add it where you catch this error in projects/admin ?